src/test/e2e/00_use_cli_test.go

```diff
@@ -70,9 +70,11 @@ func TestUseCLI(t *testing.T) {
 	pkgName := fmt.Sprintf("zarf-package-dos-games-%s.tar.zst", e2e.arch)
 
 	stdOut, stdErr, err = e2e.execZarfCommand("package", "create", "examples/game", "--confirm", "--zarf-cache", cachePath)
+	require.Contains(t, stdErr, "Creating SBOMs for 1 images")
 	require.NoError(t, err, stdOut, stdErr)
 
-	stdOut, stdErr, err = e2e.execZarfCommand("package", "inspect", pkgName)
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "inspect", pkgName, "-s")
```

Comment:
Yeah it works just fine.  I messed around with this quite a bit and what (I think) is happening is that because there is no standard in to read (we don't provide a reader to the command) it sees an EOF and the survey library we use just continues as if we had just hit enter.  The same thing will happen for other interactive commands (i.e. create without --confirm) it will give the error as if the user canceled it (not an interrupt or something else)